src/bootstrap/src/bin/rustc.rs

```diff
@@ -99,6 +99,17 @@ fn main() {
     };
     cmd.args(&args).env(dylib_path_var(), env::join_paths(&dylib_path).unwrap());
 
+    let rustc_source_dir = env::var_os("RUSTC_SOURCE_DIR").unwrap();
+    let workspace_prefix =
+        if let Ok(prefix) = env::current_dir().unwrap().strip_prefix(&rustc_source_dir) {
+            prefix.to_owned()
+        } else {
+            PathBuf::from(".")
```

Comment:
What does a `=.` remap even do? Wouldn't it make more sense to only add `--remap-path-prefix` if `strip_prefix` succeeds?

Reply:
This is a leftover from a different attempt at fixing this issue. It should indeed skip `--remap-path-prefix` in this case.